ci/_init.sh

```diff
@@ -25,10 +25,10 @@ declare -A BUILD_IMAGE_NAMES=(
 )
 export BUILD_IMAGE_NAMES
 
-declare -A TEST_IMAGE_NAMES=(
-    [$DRIVER_NAME-centos6-default]=$DOCKER_REGISTRY_NAME/client-$DRIVER_NAME-centos6-default-test:$BUILD_IMAGE_VERSION
-    [$DRIVER_NAME-centos7-node12]=$DOCKER_REGISTRY_NAME/client-$DRIVER_NAME-centos7-node12-test:$BUILD_IMAGE_VERSION
-    [$DRIVER_NAME-centos7-fips]=$DOCKER_REGISTRY_NAME/client-$DRIVER_NAME-centos7-fips-test:$BUILD_IMAGE_VERSION
-)
-export TEST_IMAGE_NAMES
+#declare -A TEST_IMAGE_NAMES=(
```

Comment:
again why we remove some block and comment a new block?

Reply:
It's the same block but commented. They are commented out since the centos images are not being run anymore due to being incompatible with the later versions of node.